Gem/Code/multiplayersample_files.cmake

```diff
@@ -89,7 +90,8 @@ set(FILES
     Source/Components/UI/UiMatchPlayerCoinCountsComponent.h
     Source/Components/UI/UiPlayerArmorComponent.cpp
     Source/Components/UI/UiPlayerArmorComponent.h
-
+    Source/Components/UI/UiRestBetweenRoundsComponent.cpp
```

Comment:
In the future, hopefully, we can put this into .Client only target instead of the shared file list we have now.

Reply:
Fixed! Added client_files.cmake